### PR TITLE
Provide default log level of info for non-configured loggers.

### PR DESCRIPTION
**Pull request by heifner**
- Modify `logger_config::update_logger` to setup a logger with the default logger if no `logging.json` is provided.
- This will make non-default loggers have a default log level of the default logger, currently that is `info`.

### CODE REVIEW

File: include/fc/log/logger.hpp [hunk 1]
```diff
@@ -21,7 +25,7 @@ namespace fc
    class logger 
    {
       public:
-         static logger get( const fc::string& name = "default");
+         static logger get( const fc::string& name = DEFAULT_LOGGER );
```
Comment (arhag) [inline include/fc/log/logger.hpp]: Also use `DEFAULT_LOGGER` in `logging_config::default_config`.

File: src/log/logger_config.cpp [hunk 2]
```diff
@@ -35,10 +35,7 @@ namespace fc {
          log = log_config::get().logger_map[name];
       } else {
          // no entry for logger, so setup with log_config's registered appenders
-         log = logger(name);
-         for( const auto& a : log_config::get().appender_map ) {
-            log.add_appender( a.second );
-         }
+         log = log_config::get().logger_map[ DEFAULT_LOGGER ];
```
Comment (arhag) [inline src/log/logger_config.cpp]: What if it does not exist because a user provided a custom logging.json that excluded a logger named `default`?

Comment (heifner) [reply src/log/logger_config.cpp]: I'm open to suggestions. Could go with what I had before in that case. Or grab first one from the map. I think I like what I had before in that case.

File: include/fc/log/logger.hpp (hunk 1 above)
Comment (heifner) [reply include/fc/log/logger.hpp]: Done.

File: src/log/logger_config.cpp (hunk 2 above)
Comment (heifner) [reply src/log/logger_config.cpp]: Went with default constructed and existing appenders if no DEFAULT_LOGGER found.

Comment (heifner) [reply src/log/logger_config.cpp]: After discussion, went with not setting up any logger if no default provided.

**Pull request merged**